File: E-Commerce-Site/E-Commerce-Site.Products/Models/SpecificProduct.cs
namespace E_Commerce_Site.Products.Models;
using System.ComponentModel.DataAnnotations.Schema;


[Table("public.products")]
public class SpecificProduct
{
    [Column("id")]
    public string Id { get; set; } = "";
    
    [Column("[title]")]
    public string Title { get; set; } = "";
    
    [Column("[description]")]
    public string Description { get; set; } = "";

    [Column("[image_path]")]
    public string ImagePath { get; set; } = "";
    
    [Column("[Retailer]")]
    public string Retailer { get; set; } = "";
    
    [Column("[cost]")]
    public decimal Cost { get; set; }
    
    [Column("[link]")]
    public string Link { get; set; } = "";
    
}

File: Python_BackEnd/Products/Models/SpecificProduct.py
import uuid
from decimal import Decimal
from pydantic import BaseModel


class SpecificProduct(BaseModel):
    id: uuid.UUID = ""
    title: str = ""
    description: str = ""
    imagePath: str = ""
    retailer: str = ""
    cost: Decimal = 0.0
    link: str = ""
    product_type: str = ""
